client/state/push-notifications/actions.js

```diff
@@ -292,8 +295,10 @@ export function toggleEnabled() {
 		} );
 		if ( enabling ) {
 			dispatch( fetchAndLoadServiceWorker() );
+			tracks.recordEvent( 'calypso_web_push_notification_enabled' );
```

Comment:
@jonathansadowski -- I think so. This would still bump the `enabled` step if someone encountered an error in the initialization process & it disabled itself, but should suffice for our purposes, I think.